gulpfile.js

```diff
@@ -223,4 +232,9 @@ gulp.task('watch', function() {
 // This task starts google app engine development server.
 // TODO(Barnabas Makonda): check if files are already generated and if so
 // do not build.
-gulp.task('start_devserver', ['build', 'gulpStartGae', 'watch']);
+var isProdMode = argv.prod_env === 'True';
+if (isProdMode) {
+  gulp.task('start_devserver', ['gulpStartGae', 'watch']);
```

Comment:
@seanlip There are two solutions:

1. Remove `build_minified_third_party_libs` and pass the output_directory as an argument through start.sh. We'll have to read the cache slug in a bash script.
But, deploy.py depends on build.py to generate all resources during a release. So, doing this is going to be a problem.

2. Move everything to gulp. This will involve much more work.